pyccel/codegen/printing/ccode.py

```diff
@@ -10,7 +10,7 @@
 from pyccel.ast.core import Assign, datatype, Variable, Import
 from pyccel.ast.core import CommentBlock, Comment
 
-
+from pyccel.ast.cwrapper import PyccelPyObject, PyArg_ParseTupleNode #PyBuildValueNode
```

Comment:
We want to avoid having cwrapper things in this file

Reply:
this is printer for those type we can put it in wrapper file   later it was just to make sure that the what is wrote in the new wrapper work .